src/common/format/common.fbs

```diff
@@ -41,6 +41,9 @@ table TaskInfo {
   actor_creation_id: string;
   // The dummy object ID of the actor creation task if this is an actor method.
   actor_creation_dummy_object_id: string;
+  // The max number of times this actor should be recontructed.
+  // If this number of 0 or negative, the actor won't be reconstructed on failure.
+  max_actor_reconstructions: int;
```

Comment:
What if you want it to be infinite?

Reply:
you can set it to the INT_MAX, which essentially means infinite. (Even if the actor reconstructs every second, it takes 136 years to exceed this number.)